projects/sigstore-go/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+RUN git clone --depth 1 https://github.com/AdamKorcz/sigstore-go --branch=fuzz-1
```

Comment:
@AdamKorcz are we cloning from your fork because fuzzers haven't been checked in yet? Same for https://github.com/google/oss-fuzz/blob/master/projects/cosign/Dockerfile#L18

Reply:
Yes. We will change the repo source once the fuzzers are merged upstream like here: https://github.com/google/oss-fuzz/pull/12378